imports/api/questions.js

```diff
@@ -291,19 +265,16 @@ Meteor.methods({
       return Meteor.call('questions.update', question)
     }
 
+    const course = Courses.findOne({_id: question.courseId })
+
     question.createdAt = new Date()
-    question.public = false
+    question.public = !course.requireApprovedQuestions
     question.creator = Meteor.userId()
 
     check(question, questionPattern)
 
     const user = Meteor.users.findOne({ _id: Meteor.userId() })
-    if (user.hasRole(ROLES.student)) {
-      // if student, can only add question to enrolled courses
-      const courses = Courses.find({ _id: { $in: (user.profile.courses || []) } }).fetch()
-      const courseIds = _(courses).pluck('_id')
-      if (question.courseId && courseIds.indexOf(question.courseId) === -1) throw Error('Can\'t add question to this course')
-    }
+    
```

Comment:
I think we still want to check that user is enrolled/instructor in the course pointed at by the courseId of the question to allow them to insert the question. Otherwise, anyone can insert questions.

 As an added safety, we should also set approved:false if the user is a student.

Reply:
Furthermore, a student should not be allowed to update a question, if the approved flag is set on that question. 
In the UI, when displaying a question from the Sidebar, don't show the EditQuestion component if the question is approved (i.e. so that the student doesn't think they can modify an approved question)